CMakeLists.txt

```diff
@@ -84,9 +84,9 @@ include(IrodsRunpathDefaults)
 
 set(IRODS_VERSION_MAJOR "4")
 set(IRODS_VERSION_MINOR "3")
-set(IRODS_VERSION_PATCH "0")
+set(IRODS_VERSION_PATCH "1")
 set(IRODS_VERSION "${IRODS_VERSION_MAJOR}.${IRODS_VERSION_MINOR}.${IRODS_VERSION_PATCH}")
-set(IRODS_PACKAGE_REVISION "1")
+set(IRODS_PACKAGE_REVISION "0")
```

Comment:
i'm not sure about this setting to 0...

Reply:
See https://github.com/irods/irods/issues/6412#issuecomment-1724656181.

I was unsure about changing that too, but that comment seems to imply this repo's package revision should be updated too.

@SwooshyCueb Should the package revision number for this repo be changed to 0 or left as 1?

Reply:
Should be changed to 0

Reply:
He has spoken.